pandas/core/arrays/floating.py

```diff
@@ -257,6 +257,15 @@ def __init__(self, values: np.ndarray, mask: np.ndarray, copy: bool = False):
             )
         super().__init__(values, mask, copy=copy)
 
+    def __neg__(self):
```

Comment:
I think @simonjayhawkins already mentioned it as well, but can you move those definitions you added here to the base class `NumericArray` in `arrays/numeric.py` instead?

Reply:
I've moved the operator definitions to `arrays/numeric.py`